docs/send-email-alerts-by-event-topic/README.md

```diff
@@ -0,0 +1,54 @@
+# Send email alerts by event topic
+
+This task allows you to receive an email alert for any event topic supported by Mechanic. Configure the task with a list of email addresses to notify, an email subject, and one or more event topics, each paired with an email body that supports liquid output tags for the main object of each event (e.g. *shopify/products/create* can use attributes like `{{ product.id }}` or `{{ product.title }}`)
+
+* View in the task library: [usemechanic.com/task/send-email-alerts-by-event-topic](https://usemechanic.com/task/send-email-alerts-by-event-topic)
+* Task JSON, for direct import: [task.json](../../tasks/send-email-alerts-by-event-topic.json)
+* Preview task code: [script.liquid](./script.liquid)
+
+## Default options
+
+```json
+{
+  "emails_to_notify__array_required": [
+    "brad@tekha.us"
+  ],
+  "general_email_subject__required": "ALERT",
+  "append_event_topic_to_email_subject__boolean": true,
+  "event_topics_and_email_bodies__keyval_multiline_required": {
+    "shopify/products/create": "Hi,\n\nProduct {{ product.id }} was created\n\nView the details in Shopify:\nOBJECT_LINK",
+    "shopify/customers/create": "Hi\n\nCustomer {{ customer.id }} was created\n\nView the details in Shopify:\nOBJECT_LINK"
+  }
+}
+```
+
+[Learn about task options in Mechanic](https://docs.usemechanic.com/article/471-task-options)
+
+## Subscriptions
+
+```liquid
+{% assign event_topics = options.event_topics_and_email_bodies__keyval_multiline_required | keys %}
+{% for event_topic in event_topics %}
+{{ event_topic }}
```

Comment:
Indent this, por favor? Two spaces in, per usual.

Alternatively, you could replace this forloop with the following:

```liquid
{{ event_topics | join: newline }}
```

Or, naturally, you could do this whole thing as a one-liner, using `keys` and `join` on the original option value.

Your pick. :D

Reply:
For some reason I had it in my head that the subscriptions box wasn't keen on white space, but I didn't bother to check prior art here to verify.